### PR TITLE
[Fix](export/outfile) Support compression when exporting data to Parquet / ORC.

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

When using Export/Outfile, we can use properties `compress_type` to specify the compression method.

Currently, Parquet file format supports `SNAPPY`, `GZIP`, `BROTLI`, `ZSTD`, `LZ4`, `LZO`, `BZ2` and `PLAIN` compression methods, default value is `SNAPPY`.

 Orc file format supports `PLAIN`, `SNAPPY`, `ZLIB`, and `ZSTD` compression methods, default value is `ZLIB`.  

Docs: https://github.com/apache/doris-website/pull/764

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run feut

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java [hunk 1]
```diff
@@ -668,7 +677,7 @@ void setParquetCompressionType(String propertyValue) {
         if (PARQUET_COMPRESSION_TYPE_MAP.containsKey(propertyValue)) {
             this.parquetCompressionType = PARQUET_COMPRESSION_TYPE_MAP.get(propertyValue);
         } else {
-            LOG.warn("not set parquet compression type or is invalid, set default to UNCOMPRESSED type.");
+            LOG.warn("not set parquet compression type or is invalid, set default to SNAPPY type.");
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java:680]:
```suggestion
            LOG.debug("not set parquet compression type or is invalid, set default to SNAPPY type.");
```

File: fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java [hunk 2]
```diff
@@ -744,6 +753,16 @@ private void getParquetProperties(Set<String> processedPropKeys) throws Analysis
     }
 
     private void getOrcProperties(Set<String> processedPropKeys) throws AnalysisException {
+        // get compression type
+        if (properties.containsKey(ORC_COMPRESSION)) {
+            if (ORC_COMPRESSION_TYPE_MAP.containsKey(properties.get(ORC_COMPRESSION))) {
+                this.orcCompressionType = ORC_COMPRESSION_TYPE_MAP.get(properties.get(ORC_COMPRESSION));
+            } else {
+                LOG.warn("not set orc compression type or is invalid, set default to ZLIB type.");
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java:761]:
```suggestion
                LOG.debug("not set orc compression type or is invalid, set default to ZLIB type.");
```

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java [hunk 3]
```diff
@@ -173,6 +174,10 @@ public class ExportJob implements Writable {
     private String withBom;
     @SerializedName("dataConsistency")
     private String dataConsistency;
+    @SerializedName("parquetCompressionType")
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java:177]:
```suggestion
    @SerializedName("pct")
```

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java [hunk 4]
```diff
@@ -173,6 +174,10 @@ public class ExportJob implements Writable {
...
+    private String parquetCompressionType;
+    @SerializedName("orcCompressionType")
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java:179]:
```suggestion
    @SerializedName("oct")
```

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — DISMISSED** [review]

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run performance

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run p0

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by wuwenchi — APPROVED** [review]

**Pull request merged**